AIPscan/Reporter/templates/report_ingest_log_tabular.html

```diff
@@ -0,0 +1,71 @@
+<!-- Ingest log tabular report. -->
+
+{% extends "report_base.html" %}
+
+{% block extra_css %}
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/style_ingest_log.css') }}">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
```

Comment:
I was anticipating the question. Have you more background on that Tessa - I have seen this suggested elsewhere but not without any particular rationale being clear. What I could half fathom was that the amount of load on production servers generated made it desirable, but I don't imagine AIPscan being used that heavily. I also believe network access to the outside world will be a requirement in the long-run with the potential requirement to access external data sources. 

But anyway, that's my understanding, some background on the why would be beneficial for me? 

I ended up not bringing it into static because it ultimately feels harder to maintain over time. And this was the pragmatic approach with very little over-head to try, i.e. this is the first report using the icons for a bit of flare so I didn't know if it would stick. That being said, I haven't a strong opinion. Though I'd like to question the norm.

Reply:
Hi Ross. FWIW I'm also not convinced of the load argument (the browser's going to have to pull that file down from somewhere to cache it, and it's probably faster some or a lot of the time doing that from a a local CDN cache near them than from our server).

I had two angles on this:

One is that it seems like we might already have (a slightly older version of) this file in our static directory: https://github.com/artefactual-labs/AIPscan/blob/main/AIPscan/static/css/fontawesome.css.

But primarily I was thinking of the issue of network access to the outside world. It's possible that I'm being too conservative here because it's been an issue for Brunnhilde in the not-too-distant past (which at one point download Bootstrap assets from GitHub until it became clear that was an issue for folks), but I guess I wouldn't be too surprised if some of our clients would prefer not to have outside network access on their Archivematica/AIPscan servers unless absolutely necessary, and from what I can tell this is the first thing in the codebase that would complicate that.

> I also believe network access to the outside world will be a requirement in the long-run with the potential requirement to access external data sources.

That's interesting, and I suspect you're ahead of me in your thinking on that.

> I ended up not bringing it into static because it ultimately feels harder to maintain over time.

I can see this as well, although I wonder if there's any reason to question in a long-term scenario whether assets may at some point no longer be available via the CDN at some point?

Honestly, this isn't a sticking point for me, so if you'd rather leave it as-is and change it down the road if someone complains, I think that would be okay.

Reply:
Thanks for the response Tessa, I'm glad I'm not quite alone in some of those thoughts. Heeding your past experiences i've tried to import it static, and it looks like it is working okay - I've wrestled with many white squares! But let me know.